### DISCUSSION TITLE
Getting match_details from games in earlier seasons

### DISCUSSION THREAD

**Discussion by GuusBikker** [community/community#129204] [Programming Help]
### Select Topic Area

Question

### Body

Hello everyone, I have been trying to use this api for one of my school projects but I can't find out how to get the matches played by a team from the 2022/2023 season. Is there anyway I could get that information using this api tool?

**Comment by 21shahvage** [comment]
you have to mention the sport and how the api wants to work.

**Reply by vatsalcshah** [reply]
To get the matches played by a team in the 2022/2023 season, you can use a sports API like Sportmonks.  After obtaining an API key, use their "Fixtures" endpoint, filtering by the team's ID and the relevant season ID. Parse the JSON response to extract match details.
